.babelrc.js

```diff
@@ -12,6 +12,7 @@ const config = {
 
 if (!process.env.BABEL_ESM) {
   config.plugins.push("@babel/transform-modules-commonjs")
+  config.plugins.push("add-module-exports")
```

Comment:
Nicke. But add the `addDefaultProperty` option, just in case :)
And you better to merge `Array#push()` calls into a single one, like so:
```js
config.plugins.push(
  "@babel/transform-modules-commonjs",
  ["babel-plugin-add-module-exports", {
    addDefaultProperty: true
  }]
)
```

Thank you 👍

Reply:
> But add the addDefaultProperty option, just in case :)

Done in ff894d9 seconds before your comment.

> And you better to merge Array#push() calls into a single one

Sure, I can do that :)

Reply:
Well, I have already merged it :D So I will make it myself x)